model/clusters_mgmt/v1/cluster_api_type.model

```diff
@@ -20,4 +20,32 @@ struct ClusterAPI {
 	URL String
 	// The listening method of the API server.
 	Listening ListeningMethod
+	
+	// Describes the access policy for CIDR Blocks to the Kubernetes API server
+	CIDRBlocksAccess CIDRBlocksAccess
+}
+
+struct CIDRBlocksAccess {
+	// Describes the CIDR Blocks allowed to access the Kubernetes API server. 
+	// Currently, only supported for ARO-HCP based clusters.
+	Allow CIDRBlocksAllowAccess
+}
+
+struct CIDRBlocksAllowAccess {
+	// There are two modes: "allow_all" and "allow_list"; if "allow_list" is provided than a non-empty 'values' list must be provided.
+	// Otherwise, if "allow_all" is provided than 'values' list should be omitted.
+	Mode CIDRBlocksAccessMode
+	// The 'values' list should contain a CIDR blocks list (An IPV4 address range in the format `<ipv4_address>/<network_mask>`). 
+	// The maximum number of CIDR blocks supported is 500. The CIDR blocks should be non-overlapping and valid.
+	// The value "0.0.0.0/0" is not considered a valid value, as the user can use "allow_all" mode to indicate this behavior.
+	Values []String
+}
+
+// CIDR Blocks Access modes.
+enum CIDRBlocksAccessMode {
```

Comment:
@miguelsorianod mentioned using enums will break the client if we introduce another value.